Library/Formula/elasticsearch.rb

```diff
@@ -128,6 +103,17 @@ def plist; <<-EOS.undent
   end
 
   test do
-    system "#{bin}/plugin", "--list"
+    system "#{bin}/plugin", "list"
+    require 'tempfile'
+    file = Tempfile.new('homebrew-elasticsearch-test')
+    file.close
```

Comment:
You can actually just write a file to the `testpath` which is in a temporary directory already.